app/controllers/proposal/collection_draft_proposals_controller.rb

```diff
@@ -226,6 +230,44 @@ def submitter_from_resource
       { name: "#{proposal_urs_user['first_name']} #{proposal_urs_user['last_name']}", email: proposal_urs_user['email_address'] } unless proposal_urs_user.blank?
     end
 
+    # Get urs users and get emails from urs for approvers
```

Comment:
This function represents the bulk of the changes between this PR and the last one.